spec/elf/Elf.c

```diff
@@ -0,0 +1,1983 @@
+
```

Comment:
Add copyright/license headers to all files in this PR

Reply:
Generated file. Can't easily add the copyright header without it getting overwritten on each regeneration of the file.

Reply:
https://github.com/microsoft/ebpf-for-windows/issues/834